### PR TITLE
upgrade examples with next.js

### PR DESCRIPTION
**Pull request by carlosstenzel**

### CODE REVIEW

**Comment by kiwicopple** [comment]
Hey @carlosstenzel - this PR seems to be doing multiple different things:

- upgrading dependencies
- running prettier
- changing translations

Perhaps we can split it into separate PR's which we can merge/comment on? I'll drop some comments now on the current changes for the new PRs

File: examples/nextjs-auth/package.json [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
-  "dependencies": {
-    "@supabase/supabase-js": "^1.2.1",
-    "next": "latest",
-    "react": "^16.13.1",
-    "react-dom": "^16.13.1",
-    "swr": "0.2.3"
-  },
-  "license": "MIT"
-}
+    "name": "with-supabase-auth",
```
Comment (kiwicopple) [inline examples/nextjs-auth/package.json:2]: It looks like you've used a code formatter which adds 4 spaces (or tabs)? We use this prettier config:
https://github.com/supabase/supabase/blob/master/.prettierrc

Is this file formatted with the correct config?

File: examples/nextjs-with-supabase-auth/package.json [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
+        "build": "next build",
+        "start": "next start"
+    },
+    "dependencies": {
+        "@supabase/supabase-js": "^1.11.9",
+        "@supabase/ui": "^0.24.2",
+        "next": "latest",
+        "react": "17.0.2",
+        "react-dom": "17.0.2",
+        "swr": "^0.5.5"
```
Comment (kiwicopple) [inline examples/nextjs-with-supabase-auth/package.json:15]: just to check - did you run the example to ensure all these versions are working?

File: i18n/README.pt-br.md [hunk 3]
```diff
@@ -23,7 +23,7 @@ Para documentação completa, visite [supabase.io/docs](https://supabase.io/docs
 ## Comunidade & Suporte
```
Comment (kiwicopple) [inline i18n/README.pt-br.md:23]: Thanks for reviewing! Should we move these to a different PR?

File: examples/nextjs-auth/package-lock.json [hunk 4]
```diff
@@ -1,12939 +0,0 @@
-{
```
Comment (kiwicopple) [inline examples/nextjs-auth/package-lock.json:1]: we should check in this lock file - we want to make sure that the example "just works" for developers using the example

**Comment by carlosstenzel** [comment]
@kiwicopple 

I will change the items. Thanks

I test it with the new dependencies and it worked perfectly.
